doc/api/fs.md

```diff
@@ -3529,7 +3529,7 @@ The kernel ignores the position argument and always appends the data to
 the end of the file.
 
 On Windows, if the file descriptor is connected to the console (e.g. `fd == 1`
-or `stdout`) a string encoded as UTF-8 or Unicode (`ucs2`) will not be rendered
+or `stdout`) a string containing non-ASCII characters will not be rendered
```

Comment:
Are you sure that, no matter which encoding you specify, no non-ASCII characters will be rendered properly by default?

Reply:
I think so, the problem is that the terminal has a non-compatible code page: practically the data arrives with the right encoding, it is the terminal that knows nothing about those. The terminal code page is determined by the Windows locale (see https://ss64.com/nt/chcp.html). From the information in that page, the above sentence seems correct.

Please suggest alternate wording if you think it's more appropriate.

Reply:
In that case the wording is appropriate, but perhaps it should be clarified that:

1. With the default code page, non-ASCII characters will not be rendered properly whether you use 'utf8', 'usc-2' or anything else.
2. With `chcp 65001`, non-ASCII characters will be rendered properly if and only if you use 'utf8'.

Reply:
Can you please suggest some alternate wording?

Reply:
```suggestion
or `stdout`) a string containing non-ASCII characters may not be rendered
```
IMHO just use `may`
It will keep this correct without getting into the minutia of this... Proper rendering depends on
1. You active codepage
2. Your `conhost.exe` version
3. The active font
4. String encoding

For example
Even a single-byte string with non-ASCII chars that matched the codepage _will_ be rendered correctly. If the codepage is available and the font has these chars.
```
> chcp 1037
> node -p "'רפאל'"
רפאל
```

Reply:
I disagree unless there is a combination of code page and encoding that renders non-ASCII characters correctly. The example is not valid since it doesn't write to the file descriptor directly, so it will work regardless of the code page.

Reply:
> I disagree unless there is a combination of code page and encoding that renders non-ASCII characters correctly.

There is, as said in the following paragraph. However, this is _not the default_ as far as we know. It's important that this is Operating System specific, so who knows if in some locale/country/configuration this is rendered correctly? I think the use of the word `may` is correct here.

@seishun can you please propose some alternate wording?

Reply:
Then, I'm going to consider this comment resolved.